packages/contract/lib/reason.js

```diff
@@ -28,25 +28,24 @@ const reason = {
       // be updated to respect (instead of computing here)
       const data = res.error.data;
       let resData;
-      if ("result" in data) {
-        // there is a single result (only in ganache 3.0+)
+      if (typeof data === "string") {
+        resData = data; // geth, Ganache >7.0.0
```

Comment:
I'm guess we don't have any tests that hit this?

Reply:
Not at present, it would seem.